### PR TITLE
Added sandbox app to samples.

### PR DESCRIPTION
**Pull request by grokys**
## What does the pull request do?

Adds a empty Sandbox app intended for experiments, repros and perf testing. We currently usually use ControlCatalog for this but that isn't ideal for a couple of reasons:

- One has to delete the XAML and then delete the codebehind to get an empty window for repros. I must have done this a million times, and _still_ I keep forgetting to delete the code in the MainWindow ctor
- It creates a bunch of items for e.g. the `ListBoxPage` that show up when doing perf testing

The Sandbox app is intended to be left empty and used in these situations.

### CODE REVIEW

File: samples/Sandbox/Sandbox.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
```
Comment (rstm-sf) [inline samples/Sandbox/Sandbox.csproj:4]: The sandbox thus becomes platform dependent

Comment (FrankenApps) [reply samples/Sandbox/Sandbox.csproj:4]: Are you sure? I think this only means hiding the console window in .netcore 3+?

Comment (rstm-sf) [reply samples/Sandbox/Sandbox.csproj:4]: Oops, yeah. I misinterpreted ([1](https://github.com/dotnet/sdk/blob/afe72f4eca6065e945705293cb6b33a31fa1dc50/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets#L36), [2](https://github.com/dotnet/sdk/blob/afe72f4eca6065e945705293cb6b33a31fa1dc50/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L457)). 'WinExe' should be interpreted as something like 'non-console EXE'...

Comment (kekekeks) [reply samples/Sandbox/Sandbox.csproj:4]: That property controls the `Subsystem` field of `IMAGE_OPTIONAL_HEADER` of PE executable. .NET images always come in PE executable format.

**Review by danwalmsley — APPROVED** [review]
LGTM

**Pull request merged**